middleware/cache/cache.go

```diff
@@ -96,6 +109,9 @@ func New(config ...Config) fiber.Handler {
 			if len(e.cencoding) > 0 {
 				c.Response().Header.SetBytesV(fiber.HeaderContentEncoding, e.cencoding)
 			}
+			for k, v := range e.headers {
```

Comment:
check if the element is not nil

important, so that we save the allocations for the map when the switch is deactivated